content/en/blog/news/2022-gsoc-work-report.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "GSoC 2022 Final Report: Building Jenkins X UI"
+date: 2022-11-13
+draft: false
+description: This project replaces the older UI with the new UI with more powerful features.
+categories: [blog]
+keywords: [Community, jenkins-x, gsoc, 2022]
+slug: "GSoC-2022-work-report"
+aliases: []
+author: Rajat Gupta
+---
+
+## Jenkins X New UI
+It is a web application built with [Golang](https://go.dev/) for the backend and [Sveltekit](https://kit.svelte.dev/) for the frontend, both of which are built together and used in the same container. To function properly, it must be installed as a helm chart with Jenkins X CRDs.
```

Comment:
Would be great to follow this: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line